eknrenderer/eknrenderer/eknr-renderer.c

```diff
@@ -0,0 +1,1311 @@
+/* Copyright 2018 Endless Mobile, Inc. */
+
+#include <string.h>
+#include <stdlib.h>
+
+#include <endless/endless.h>
+#include <glib/gi18n.h>
+#include <json-glib/json-glib.h>
+#include <mustache.h>
+#include <libintl.h>
```

Comment:
Alphabetical order. But not sure this include is actually needed.

Reply:
It probably is not since we include `gi18n` now